content/projects/gsoc/2023/project-ideas/displaying_plugin_health_scores.adoc

```diff
@@ -8,9 +8,8 @@ status: published
 sig: platform
 skills:
 - Java
-- YAML
-- Command line tools
-- Package management tool theory
+- Data Analysis
+- Data Presentation
```

Comment:
I think it would be a nice idea to mention the technologies used in this project maybe?
Since it largely deals with displaying health scores on plugins.jenkins.io and also playing with the frontend of [plugin-health.jenkins.io](https://plugin-health.jenkins.io/), so maybe: `ReactJS, webpack`... What do we think?

Reply:
Sounds like a decent idea. I have no issues with that

Reply:
Done, changed "Data Presentation" to "Data Presentation using React.js, webpack, and other frontend technologies"